gammapy/irf/psf/tests/test_table.py

```diff
@@ -12,6 +14,21 @@ def psf_3d():
     return PSF3D.read(filename, hdu="PSF")
 
 
+def test_psf_3d_wrong_units():
+    energy_axis = MapAxis.from_energy_edges([80, 125] * u.GeV, name="energy_true")
+
+    offset_axis = MapAxis.from_edges([0, 1, 2], unit="deg", name="offset")
+
+    rad_axis = MapAxis.from_edges([0, 1, 2], unit="deg", name="rad")
+
+    wrong_unit = u.cm**2 * u.s
+    data = np.ones((energy_axis.nbin, offset_axis.nbin, rad_axis.nbin)) * wrong_unit
+    with pytest.raises(ValueError) as error:
+        psf3d = PSF3D(axes=[energy_axis, offset_axis, rad_axis],
+                 data=data)
+        assert error == (f"Error: {wrong_unit} is not an allowed unit. {psf3d.tag} requires dimensionless data quantities!")
```

Comment:
is it dimensionless or per `sr`?

Reply:
No, a psf 3d should have unit `sr^-1`, https://gamma-astro-data-formats.readthedocs.io/en/latest/irfs/full_enclosure/psf/psf_table/index.html